src/index.js

```diff
@@ -12,7 +12,30 @@ if (!app.requestSingleInstanceLock()) {
   )
 
   // No windows were opened at this time so we don't need to do app.quit()
-  process.exit(0)
+  process.exit(1)
+}
+
+function handleError (e) {
+  dialog.showMessageBox({
+    type: 'error',
+    title: 'Something wrong happened',
+    message: e.stack + '\nPlease choose one of the options bellow. All of them will copy the error message to the clipboard.',
+    buttons: [
+      'Close',
+      'Open logs',
+      'Create a new issue'
+    ]
+  }, option => {
+    if (option === 1) {
+      const path = app.getPath('userData')
+      shell.openItem(path)
+    } else if (option === 2) {
+      let text = 'Please describe what you were doing when this error happened.\n\n```\n' + e.stack + '\n```'
```

Comment:
@lidel I added OS and IPFS Desktop version. Electron and Go IPFS version can be deduced by IPFS Desktop version for now.

Reply:
I am worried we will forget about this and when backend switcher is added it will be to late :) 

Also, better to have more data than too little. 
It may be good to know what Chrome version is behind Electron too:
- [`process.versions.electron`](https://electronjs.org/docs/api/process#processversionselectron)
- [`process.versions.chrome`](https://electronjs.org/docs/api/process#processversionschrome)

Reply:
Done 😄